sites/docs/src/content/docs/guidelines/pipelines/recommendations/publication_credit.md

```diff
@@ -16,7 +16,7 @@ All nf-core pipeline publications and presentations should cite [The nf-core fra
 
 **Optionally**, the nf-core community can be included as a consortium co-author. This option is preferred when a pipeline has made extensive use of existing nf-core pipeline components (e.g., modules and subworkflows) written by other community members who were not directly involved in the pipeline itself. For example, see hgtseq: [A Standard Pipeline to Study Horizontal Gene Transfer](https://doi.org/10.3390/ijms232314512).
 
-If members of the nf-core community have provided significant input to the creation or maintenance of a pipeline, please consider adding them as coauthors on the pipeline publication or presentation.
+If members of the nf-core community have provided significant input to the creation or maintenance of a pipeline, please consider adding them as coauthors on the pipeline publication or presentation. We also recommend transparency on working on a piece where the pipeline is a major part of it. If you plan to write a paper about a pipeline, please share this with the community on the pipeline channel on the nf-core Slack workspace so that other contributors can join your effort.
```

Comment:
```suggestion
If members of the nf-core community have provided significant input to the creation or maintenance of a pipeline, please consider adding them as coauthors on the pipeline publication or presentation. We also recommend transparency when working on a project where the pipeline plays a significant role. If you plan to write a paper about a pipeline, please share this information with the community in the pipeline specific channel on the nf-core Slack workspace, so that other contributors can join and support your effort.
```

Reply:
I would go even further 

```suggestion
If members of the nf-core community have provided significant input to the creation or maintenance of a pipeline, please consider adding them as coauthors on the pipeline publication or presentation.

We also strongly recommend transparency when working on a project where the pipeline plays a significant role. If you plan to write a paper about a pipeline, please share this information prior and during writing of the manuscript with the community in the pipeline specific channel on the nf-core Slack workspace, so that other major contributors can join and support your effort.
```

I sort of wonder if we shiould put a caveat somewhere (cc' @ewels maybe/) that nf-core provides 'no warranty to the generation of publication' etc or something... or some wording saying while we want to facilitate openness and collabration, we are not responsible for authorship etc.

Reply:
I wonder if we should show leadership here and define some criteria. We (nf-core) don't want to end up being mediators for authorship, but we could be more proactive in defining criteria.

[For example](https://www.icmje.org/recommendations/browse/roles-and-responsibilities/defining-the-role-of-authors-and-contributors.html#two):

The ICMJE recommends that authorship be based on the following 4 criteria:
- Substantial contributions to the conception or design of the work; or the acquisition, analysis, or interpretation of data for the work; AND
- Drafting the work or reviewing it critically for important intellectual content; AND
- Final approval of the version to be published; AND
- Agreement to be accountable for all aspects of the work in ensuring that questions related to the accuracy or integrity of any part of the work are appropriately investigated and resolved.

These don't really apply here, but something similar might help in being proactive rather than reactive when deciding what constitutes a contribution.

Reply:
I agree with all your points! I will commit Sateesh/James' contribution, but we can keep discussing how to make it even better, based on Chris' comment.

Reply:
I dunno, I don't think we should be proactive from the point of:

1. in reality we aren't an entity that has any weight (we can't get lawyers involved, and yes that can happen), so can we really have any say in such things? Who is then responsible?
2. co-authorship arguments are really political and messy and we can't expect us as volunteers involved to get long drawn out arguments
3. many different co-authorship guidelines do exist (e.g. the ICMJE thing you quote), this varies per field AND is often not actually interpreted in the same way, or just out-right ignored and say 'well we've never done that before.

However I might be biased and burnt out because I've been too long in a field where these discussions  and blown up waaaaay too much and I'm sick of it :sweat_smile: